docs/olm.md

```diff
@@ -0,0 +1,26 @@
+# OLM
+
+We currently support a resource which wraps arbitrary OLM packages.
+
+## Vanilla Kubernetes
+
+For a fresh Kubernetes cluster (e.g., `kind`), you can install the OLM by running the following command: `curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.17.0/install.sh | bash -s v0.17.0`
+
+You can see installable operators by running the following command: `kubectl get packagemanifests -n olm`, which displays all the available packagemanifests, each of which can correspond to one or more subscription.
+
+Prior to creating a though subscription, you will need to create an operator-group, there are plans to include this automatic reconsilation for operator groups in the lifecycle for Operator resources in the future.
+
+## Openshift
+
+For an Openshift cluster, the OLM will be installed already, and you can see available operators by running the following command: `kubectl get packagemanifests -n openshift-marketplace`. By default there should be a valid operator group in the default namespace.
```

Comment:
command in code block


or an Openshift cluster, the OLM will be installed already, and you can see available operators by running the following command:

```
kubectl get packagemanifests -n openshift-marketplace
```
By default there should be a valid operator group in the default namespace.